legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-postgres/src/test/java/org/finos/legend/engine/persistence/components/e2e/versioning/TestDedupAndVersioning.java

```diff
@@ -0,0 +1,830 @@
+// Copyright 2024 Goldman Sachs
```

Comment:
After changing the above methods to use the ones in TestUtils and BaseTest, we dont need to duplicate this file here, and in turn we can delete the data files too, to keep it cleaner